.github/workflows/publish-flutter.yml

```diff
@@ -61,12 +61,17 @@ jobs:
           cp ../breez-sdk/libs/sdk-flutter/pubspec.yaml .
           cp ../breez-sdk/libs/sdk-flutter/pubspec.lock .
 
+      - uses: actions/download-artifact@v3
```

Comment:
@roeierez Here's where the source file is downloaded

Reply:
Ah.. I see, makes sense.
I agree with this approach.